src/login/InitResetForm/InitResetForm_actions.js

```diff
@@ -0,0 +1,23 @@
+export const DEAL_WITH_EMAIL = "DEAL_WITH_EMAIL";
+export const POST_RESET_PASSWORD_RESET_SUCCESS = "POST_RESET_PASSWORD_RESET_SUCCESS"
```

Comment:
does this action refer to sending the email address to the backend (to generate a link in an email). If so, is it possible to name it accordingly?

Reply:
This action type comes from the backend, and signals a successful operation on our request.